edgraph/server.go

```diff
@@ -665,15 +680,16 @@ func doQueryInUpsert(ctx context.Context, req *api.Request, gmu *gql.Mutation) (
 		}
 	}
 
-	updateMutations(gmu, varToUID)
+	updateUIDinMutations(gmu, varToUID)
+	updateVALinMutation(gmu, varToVAL)
```

Comment:
could just be `updateValInMutations` and `updateUIDInMutations` (notice the `s` in both)